lineman/app/models/group_records_interface.coffee

```diff
@@ -6,3 +6,6 @@ angular.module('loomioApp').factory 'GroupRecordsInterface', (BaseRecordsInterfa
       @fetch
         path: "#{parentGroup.id}/subgroups"
         cacheKey: "subgroupsFor#{parentGroup.key}"
+
+    uploadPhoto: (group, file, kind) =>
```

Comment:
Cool, this is a merge artifact I think; the uploadPhoto method should be on group_model now.